### PR TITLE
error_pages

### PR DESCRIPTION
**Pull request by n0b0dy-su**
# Description
I create the error pages views and their configuration

### CODE REVIEW

File: app/helpers/errors_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ErrorsHelper
```
Comment (OchoaHub) [inline app/helpers/errors_helper.rb:1]: If you don't need helpers, delete them

File: app/views/errors/internal_server_error.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div id="notfound" style="text-align: center;">
+  <div class="notfound">
+    <div class="notfound-404" style="color: #90E0EF;font-weigh: bold">
+      <h3>Sorry we have an internal proble</h3>
```
Comment (OchoaHub) [inline app/views/errors/internal_server_error.html.erb:4]: Problem*

**Review by OchoaHub — APPROVED** [review]
Looks good in general

**Review by LDVera — APPROVED** [review]
Good job :D

**Review by jnegrete15 — DISMISSED** [review]

**Review by LDVera — APPROVED** [review]

**Review by OchoaHub — APPROVED** [review]

**Review by jnegrete15 — APPROVED** [review]

**Pull request merged**